cmd/vicadmin/server.go

```diff
@@ -120,6 +115,7 @@ func (s *server) serve() error {
 
 	s.mux.Handle("/css/", http.StripPrefix("/css/", http.FileServer(http.Dir("css/"))))
 	s.mux.Handle("/images/", http.StripPrefix("/images/", http.FileServer(http.Dir("images/"))))
+	s.mux.Handle("/fonts/", http.StripPrefix("/fonts/", http.FileServer(http.Dir("fonts/"))))
```

Comment:
Can't we use some other font or something else (like words with colors, Success/Fail etc.)?